iceoryx_hoofs/include/iceoryx_hoofs/internal/concurrent/lockfree_queue/resizeable_lockfree_queue.inl

```diff
@@ -124,7 +124,7 @@ uint64_t ResizeableLockFreeQueue<ElementType, MaxCapacity>::decreaseCapacity(con
     uint64_t decreased = 0U;
     while (decreased < toDecrease)
     {
-        BufferIndex index;
+        BufferIndex index{0};
```

Comment:
This fixes the nightly clang-tidy run: https://github.com/eclipse-iceoryx/iceoryx/actions/runs/3576692744